src/main/java/core/basesyntax/garage/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.garage;
+
+public class Excavator extends Machine{
+    public void doWork() {
+        System.out.println("Excavator started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working");
```

Comment:
The method 'stopWork' should provide an informative message indicating that it is an 'Excavator' that has stopped working. Consider including the type of machine in the message.